### PR TITLE
Document Name Server Sets

### PR DESCRIPTION
**Pull request by lokst**
Document the upcoming Name Server Sets feature

### Tasks
- [x] Add article for name server sets: https://deploy-preview-968--dnsimple-support.netlify.app/articles/name-server-sets/
- [x] Add article for zone NS records: https://deploy-preview-968--dnsimple-support.netlify.app/articles/zone-ns-records/
- [x] Update domain delegation articles
  - [x] [Setting Name Servers](https://deploy-preview-968--dnsimple-support.netlify.app/articles/setting-name-servers/)
  - [x] [Delegating a Domain registered with DNSimple to DNSimple](https://deploy-preview-968--dnsimple-support.netlify.app/articles/delegating-dnsimple-registered/)
- [x] Update [secondary DNS](https://deploy-preview-968--dnsimple-support.netlify.app/articles/secondary-dns/) article
- [x] Optimize images
- [x] Seek reviews

### Prerequisites
- [x] Deploy https://github.com/dnsimple/dnsimple-app/pull/20998
- [x] Deploy https://github.com/dnsimple/dnsimple-app/pull/21005
- [x] Deploy https://github.com/dnsimple/dnsimple-app/pull/21041

### CODE REVIEW

**Review by kojdm — APPROVED** [review]
LGTM!

File: content/articles/name-server-sets.markdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    ![](/files/name-server-sets-from-account.png)
+
+1. Click on <label>Add</label> to create a new name server set.
+
+    ![](/files/name-server-sets-add.png)
+
+1. Enter a unique label for the new name server set, and the hostname and optionally, glue IP(s), for each name server that you would like to be in the name server set. Glue IP(s) are used only when changing [name server delegation](/articles/setting-name-servers) for domains registered with DNSimple.
+
+    ![](/files/name-server-sets-new.png)
```
Comment (dallasread) [inline content/articles/name-server-sets.markdown:45]: Should we add alt text for images?

**Review by dallasread — APPROVED** [review]
This is really great! Thanks for doing this!

File: content/articles/name-server-sets.markdown [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+## Types of Name Server Sets
+
+There are 2 types of name server sets:
+- System name server sets
+- Account name server sets
+
+### System Name Server Sets {#system-name-server-sets}
+
+System name server sets are pre-defined name server sets provided by DNSimple which cannot be modified. They contain [DNSimple name servers](/articles/dnsimple-nameservers).
```
[the remark this answers is not in the document]
Comment (lokst) [reply content/articles/name-server-sets.markdown]: Added a note 🙂

**Comment by lokst** [comment]
> I'd like to tighten the copy here, for the sake of brevity and clarity in the instructions, to:

@itsalyse Great suggestion, I've made the change 🙂

**Comment by dallasread** [comment]
We have the videos we can drop in now.

**Comment by dallasread** [comment]
If we decide to plan-restrict the feature, we need to update it here.

**Pull request merged**